new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiDomain.java

```diff
@@ -757,4 +767,23 @@ private void checkServiceCreated(String serviceName, String domNamespace) {
 
   }
 
+  private void checkServerReadyStatusByExec(String podName, String namespace) {
+    final V1Pod pod = assertDoesNotThrow(() -> oracle.weblogic.kubernetes.assertions.impl.Kubernetes
```

Comment:
In general we would not want the test code directly works with low-level Kubernetes objects. But I am not worried about this here because this method is a sample to demonstrate and test the new functionality, which eventually may not needed.

Reply:
Yes, I used the low-level Kubernetes call because I know that Pani has a PR that has an action to get the Pod.  I'll change it to use the action in Pani's PR when it eventually gets merged or we can remove this example altogether. IMO we should consider consolidating the Kubernetes API's into one Kubernetes class from which both actions and assertions would then share.

Reply:
sounds good.